tests/e2e_logging.py

```diff
@@ -1336,8 +1354,8 @@ def additional_interfaces(local_node_id):
         if args.package == "samples/apps/logging/liblogging":
             network = test_receipts(network, args)
             network = test_historical_query_sparse(network, args)
-        network = test_historical_receipts(network, args)
         if "v8" not in args.package:
+            network = test_historical_receipts(network, args)
```

Comment:
Do you know why this test no longer works in v8?

Reply:
Yes, it's because it now requires leaf_components to be exposed so the commit_evidence is available. I will do that next, but it's not a requirement for 2.0.